ynput/OpenPype
# Chore: Substance Painter Addons for Ayon

## Changelog Description
Substance Painter Addons for Ayon 
![image](https://github.com/ynput/OpenPype/assets/64118225/d24c8a92-cf1c-42eb-95d5-bafeffdd5fe6)

## Additional info
n/a

## Testing notes:
1. use the script to create server addon
2. Install your addon into your ayon server
3. If you are in dev mode, create server addon with unzipped and links the directory where you unzipped the addon in ayon_docker
4. Create new bundle with the latest version of application, openpype and substance painter addons.
5. Set Production
6. Go to Project Settings-> Your Project-> Attributes->Applications
7. Add Substance Painter
8. Run ayon launcher with the latest version of addons
9. When you check the launcher in your task you would find substance painter
